packages/make-styles/src/makeStyles.test.ts

```diff
@@ -171,4 +173,84 @@ describe('makeStyles', () => {
       }
     `);
   });
+  it('handles tokens', () => {
+    const rendererA = createDOMRenderer(createFakeDocument());
+    const rendererB = createDOMRenderer(createFakeDocument());
```

Comment:
Is there sense to test this with multiple renderers? Or it's just a copy-paste from a previous test case?

Reply:
Is just copy paste, I'll remove it